battery-calcuator/index.html

```diff
@@ -0,0 +1,508 @@
+
+            <div class="bc-row">
+                <div class="bc-label">
+                    <p class="bc-label-main">Enter distance between topside and seabed modem (m)</p>
+                    <p class="bc-label-sub">(Slant Range)</p>
+                </div>
+                <div class="bc-input">
+                    <input v-model="distanceInput" type="number" id="range" min="1" max="1500" maxlength="4">
+                </div>
+                <div class="bc-unit">m</div>
+
+                <div class="error">[[ rangeError ]]</div>
+            </div>
+
+            <div class="bc-row">
+                <div class="bc-label">
+                    <p class="bc-label-main">Select the appropriate series of Acoustic Smart Modem to be used</p>
+                </div>
+                <div class="bc-input bc-column radio">
+                    <label>
+                        <input v-model="recommended" type="radio" id="hf-input" value="hf" :disabled="distanceInput > 150">
+                        S60H series modems
+                    </label>
+                    <label>
+                        <input v-model="recommended" type="radio" id="mf-input" value="mf" >
+                        M25M series modems
+                    </label>
+                </div>
+            </div>
+
+            <div class="bc-row">
+                <div class="bc-label">
+                    <p class="bc-label-main">Enter file size for each transmission</p>
+                    <p class="bc-label-sub">(Use the filesize (in kilobytes) obtained from the ADCP software)</p>
+                </div>
+                <div class="bc-input">
+                    <input v-model="filesizeInput" type="number" id="file-size" min="0" max="1500">
+                </div>
+                <div class="bc-unit">kB</div>
+                <div class="error">[[ filesizeError ]]</div>
+            </div>
+
+            <div class="bc-row">
+                <div class="bc-label">
+                    <p class="bc-label-main">Are you planning for daily or  weekly or monthly download?</p>
+                </div>
+                <div class="bc-input radio">
+                    <label>
+                        <input v-model="downloadFrequency" type="radio" id="frequency-daily" value="1" >
+                        Daily
+                    </label>
+                    <label>
+                        <input v-model="downloadFrequency" type="radio" id="frequency-weekly" value="7" >
+                        Weekly
+                    </label>
+                    <label>
+                        <input v-model="downloadFrequency" type="radio" id="frequency-monthly" value="30" >
+                        Monthly
+                    </label>
+                </div>
+            </div>
+
+            <div class="bc-row">
+                <div class="bc-label">
+                    <p class="bc-label-main">How many times a day you need to transmit in the chosen duration?</p>
```

Comment:
> I had a hard time understanding the meaning of this field. Is this an accurate reformulation of the question? (Just asking for clarification, not suggesting this as an alternative.)
> 
> > On each day that you visit, how many time do you have to download a file of the file size given above?

@ettersi You are right. We will look for better wordings :)

Reply:
Just a thought, maybe instead of asking user for file size and number of transmissions, we could just ask for overall download size? Basically that would amount to having the user do this part of the computation, so it's a tradeoff between putting more work on the user and giving them more flexibility.